### ISSUE TITLE
GPU support?

### ISSUE THREAD

**Issue by kwhitley** [mvrahden/recurrent-js#2]
Where's the library for recurrent-js-gpu? (from NPM)

I'm very curious to see the GPU bindings, as this library appears to have no real deps, nor does the README match up with the 1.4.9 on NPM.

Sidenote: Any docs on implementing/training/testing an LSTM with the GPU-enabled library?

Cheers

**Comment by mvrahden** [comment]
Hey @kwhitley,
sorry for the late response, I was handicapped the last two month.
Yeah the GPU version was published too early and is still hanging in a private repo, since I couldn't confirm whether the implementation was working out.
I'm also still figuring out how to keep the common code of both repos in sync while supporting the GPU version on the other hand.
A further plan is, to deliver some tests for the matrix and some other utility functions, so one could assure that these functions are implemented correctly not only by the recurrent-js, but also by its gpu version.
I haven't worked with the LSTM yet, since till now I just finished the refactoring of the code to reusable components. But all the necessary utilities should be available.
I invited you to both repositories. Feel free to contribute :)
